infrastructure/iac/types.go

```diff
@@ -40,3 +40,17 @@ type iacIssue struct {
 	Documentation  lsp.Uri        `json:"documentation"`
 	IacDescription iacDescription `json:"iacDescription"`
 }
+
+type IssueData struct {
+	Id            string   `json:"id"`
```

Comment:
- `Id` exists already on the parent type `ScanIssue`, thus no need to have it declared as "additional data"
- `Title` exists already on the parent type `ScanIssue`, is it any different to the one there? It could be, but please verify and perhaps name it differently.
- `Severity` exists already on the parent type `ScanIssue`

Reply:
The title should not be any different, but not sure how to verify?